package.json

```diff
@@ -6,6 +6,7 @@
     "clean": "rimraf .cache && rimraf coverage && rimraf dist && rimraf node_modules",
     "prebuild": "rimraf dist",
     "build": "rm -rf dist && tsc --noEmit && parcel build index.html --public-url ./",
+    "buildOnWindows": "rimraf dist && tsc --noEmit && parcel build index.html --public-url ./",
```

Comment:
jak dla mnie wystarczy to podmienic na build i bedzie dzialac wszedzie. Nie wiem skad sie rm -rf wzielo. rimraf wlasnie po to jest.

Reply:
a jak Ty masz maca to rimraf u Cb zadziała?
Bo coś czytałem, że rimraf jest na windowsa, a rm -rf na linuxa i  maca ... jeśli tak, to czy wtedy na githubie się zbuduje projekt, skoro on jest na Linuxie

Reply:
Jak dla mnie rimraf dziala wszedzie, bo to polecenie z node. rm -rf nie dziala na Windows, bo tam czegos takiego nie ma.